docs/api-table.js

```diff
@@ -1412,7 +1419,7 @@ export const elements = {
         '',
         'Will make the expandable full-width on sm-size',
       ],
-      ['info', 'boolean', 'false', 'TStyles the box with a color'],
+      ['~~info~~', 'boolean', 'false', 'TStyles the box with a color. @deprecated Do not use.'],
```

Comment:
If this attribute is removed in v2, should we have it documented as deprecated or should it be removed completely? I always thought marking something as deprecated means it's still available but might/will be removed.

Reply:
Yes that makes sense. In that case I will remove the `info` prop from the props-table. But is it enough inform in a blog post for when we have made a stable release that the `info` prop has been removed from the Expandable component? Or should we also inform this on the Expandable doc's page?

Reply:
As a user who was relying on this prop, I'd expect this information to be available in the release notes/changelog for 2.0.0. Otherwise I believe if someone starts to use this component at this stage, they don't need to know what props no longer are in use.

Reply:
Removed `info` prop in this commit: [46b0ea3](https://github.com/warp-ds/tech-docs/pull/216/commits/46b0ea31f3d0f4c19a8d419a324d6eba755068da)

Reply:
And if there is a blog post that lists all breaking changes from v2.0.0, then I'd include a note on that there, too.